wherehows-web/app/components/dataset-compliance.ts

```diff
@@ -1401,13 +1405,13 @@ export default class DatasetCompliance extends Component {
      * @returns {(Promise<void>)}
      */
     didEditPurgePolicy(this: DatasetCompliance): Promise<void> {
-      const { complianceType = null } = get(this, 'complianceInfo') || {};
+      const retentionPolicy = get(this, 'retentionPolicy');
 
-      if (!complianceType) {
+      if (!retentionPolicy || !retentionPolicy.purgeType) {
         return this.needsPurgePolicyType();
       }
 
-      if (isExempt(complianceType)) {
+      if (isExempt(<PurgePolicy>retentionPolicy.purgeType)) {
```

Comment:
Probably safer to check that `retentionPolicy` is not `null` before reading the `purgeType` instead of an assertion